### ISSUE TITLE
Host header replaced in 1.0.0-RC7

### ISSUE THREAD

**Issue by milenkovicm** [kamon-io/kamon-akka-http#27]
Hi guys, I'm testing `1.0.0-RC7` which is overall very good, but I've spotted small issue with akka-http module. 

If custom host header is provided with http request it wont be respected. Custom header has been used to route the request in the service mash we use.

Issue is simple to reproduce:

```scala
  val request = HttpRequest(uri = "http://localhost:8090/", headers = List[HttpHeader](Host("some.header")))
  Http().singleRequest(request).foreach(println)
```

with `kamon-akka-http-2.5` will produce 

```
List(Host: localhost:8090, X-B3-TraceId: 1c79f68f94675332, X-B3-Sampled: 0, X-B3-ParentSpanId: , X-B3-SpanId: ab7098e5cf342606, User-Agent: akka-http/10.0.11, Timeout-Access: <function1>)
```
on server side. 

If `kamon-akka-http-2.5` is removed from class path, we get expected result 
```
List(Host: some.header, User-Agent: akka-http/10.0.11, Timeout-Access: <function1>)
```
`Host` header as expected.

I've tried to pin-point peace of code responsible for header replacement but could not find it.

**Comment by milenkovicm** [comment]
duplicate of #25,
tested with `1.0.0-RC8-c453caa2b53b78ac70c5ebb3c06cf20f680899b0` works as expected

**Comment by ivantopo** [comment]
Hey @milenkovicm, awesome! You can actually try `1.0.0` already :smile:, we published the artifacts last night and are about to announce it.

**Comment by milenkovicm** [comment]
I've tried it, 
I'm getting error with `io.kamon#kamon-scala-future_2.12;1.0.0`, it is not synced with maven central. I don't reference bintray release repo  

```
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  ::          UNRESOLVED DEPENDENCIES         ::
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn]  :: io.kamon#kamon-scala-future_2.12;1.0.0: not found
[warn]  ::::::::::::::::::::::::::::::::::::::::::::::
[warn] 
[warn]  Note: Unresolved dependencies path:
[warn]          io.kamon:kamon-scala-future_2.12:1.0.0 
[warn]            +- io.kamon:kamon-akka-2.5_2.12:1.0.0
[error] sbt.librarymanagement.ResolveException: unresolved dependency: io.kamon#kamon-scala-future_2.12;1.0.0: not found
```
let me know when to re-try

**Comment by ivantopo** [comment]
ouch! forgot to sync that repo with maven central. You can add our [bintray releases repo](https://bintray.com/kamon-io/releases) to your build or just wait a bit, artifacts are on its way!

**Comment by milenkovicm** [comment]
no hurry, will test it from maven central

**Comment by milenkovicm** [comment]
All green now.

deps i have are:

```
      "io.kamon"    %% "kamon-akka-http-2.5"   % "1.0.0",
      "io.kamon"    %% "kamon-system-metrics"  % "1.0.0",
      "io.kamon"    %% "kamon-akka-remote-2.5" % "1.0.0",
      "io.kamon"    %% "kamon-akka-2.5"        % "1.0.0",
      "io.kamon"    %% "kamon-jaeger"          % "1.0.0",
      "io.kamon"    %% "kamon-prometheus"      % "1.0.0",
```

will let you know if i notice any regressions

**Comment by ivantopo** [comment]
awesome!

**Comment by ivantopo** [comment]
btw, I just published kamon-jaeger 1.0.1.. there was a small inconsistency in the reporter name, it changed from `Jaeger` to `JaegerReporter`.. sadly I only catched that after I started updating the docs.

**Comment by milenkovicm** [comment]
cool, will update to it, once it shows up in maven central
